explorer/app/components/Project/common/projectTransformer.ts

```diff
@@ -396,6 +396,7 @@ function mapPublications(
       doi: publicationResponse.doi ?? "", // Maps any null value to string.
       officialHcaPublication:
         publicationResponse.officialHcaPublication ?? false, // Maps any null value to "false".
+      publicationUrl: publicationResponse.publicationUrl ?? "",
```

Comment:
Can you tell me why this was added?

Reply:
I was getting an error from lint cuz the interface was expecting a string and the response could be undefined. It seems that was fixed. I'll remove this.

Reply:
Thanks Bruce! It's possible for a project to not have a publication URL so your handling here is spot on.

Reply:
@BruceRodrigues, I've created a ticket covering the updates required in the corresponding component here: #186.